Native method getUptime not implemented

Trying to run some code on doppiojvm and getting this exception:

```
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at doppio.JarLauncher.main(JarLauncher.java:34)
Caused by: java.lang.UnsatisfiedLinkError: Native method 'sun.management.VMManagementImpl.getUptime0()J' not implemented.
Please fix or file a bug at https://github.com/plasma-umass/doppio/issues
	at sun.management.VMManagementImpl.getUptime0(Native Method)
	at sun.management.VMManagementImpl.getUptime(VMManagementImpl.java:183)
	at sun.management.RuntimeImpl.getUptime(RuntimeImpl.java:113)
	at org.eclipse.jetty.util.log.Log.initialized(Log.java:170)
	at org.eclipse.jetty.util.log.Log.getLogger(Log.java:290)
	at org.eclipse.jetty.util.log.Log.getLogger(Log.java:280)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.<clinit>(AbstractLifeCycle.java:35)
	... 5 more
```

--- a/src/natives/sun_management.ts
+++ b/src/natives/sun_management.ts
@@ -94,6 +94,10 @@ export default function (): any {
       return Long.fromNumber(thread.getJVM().getStartupTime().getTime());
     }
 
+    public static 'getUptime0()J'(thread: JVMThread, javaThis: JVMTypes.sun_management_VMManagementImpl): Long {
+      return Long.fromNumber(Date.now() - thread.getJVM().getStartupTime().getTime());
+    }
+
     public static 'getAvailableProcessors()I'(thread: JVMThread, javaThis: JVMTypes.sun_management_VMManagementImpl): number {
       return 1;
     }

